File: NCore/Commands/Commanditems.cs
using System;
using System.Threading.Tasks;
using NCore;

namespace NCore
{
    public class Commanditems : Command
    {
        public Commanditems() : base("items", NCore.GetNCore().lang.get("commands.items.description"), "netcraft.command.items", "items")
        {
        }

        public override async Task<bool> OnCommand(CommandSender sender, Command cmd, string[] args, string label)
        {
            if (args.Length == 0)
            {
                var a = Enum.GetNames(typeof(Material));
                await sender.SendMessage(string.Join(", ", a).ToLower());
                return true;
            }

            return false;
        }
    }
}

File: NetcraftServer/Commands/Commanditems.vb
Imports NetcraftServer

Public Class Commanditems
    Inherits Command
    Sub New()
        MyBase.New("items", "Показывает список вещей в игре", "items")
    End Sub
    Public Overrides Function OnCommand(sender As CommandSender, cmd As Command, args() As String, label As String) As Boolean
        If args.Length = 0 Then
            Dim a = [Enum].GetNames(GetType(Material))
            sender.SendMessage(String.Join(", ", a).ToLower)
            Return True
        End If
        Return False
    End Function
End Class
